INSTALL_DOCKER.rst

```diff
@@ -0,0 +1,259 @@
+
+Check QEMU-lite
+~~~~~~~~~~~~~~~
+
+`Clear Containers`_ uses an optimised version of `QEMU`_ called `QEMU-lite`_
+You can now check that the `QEMU-lite`_ package is installed and functioning:
+
+  ::
+
+    # qemu-lite-system-x86_64 --version
+    QEMU emulator version 2.6.0, Copyright (c) 2003-2008 Fabrice Bellard
+
+    # qemu-lite-system-x86_64 --machine help | grep pc-lite
+    pc-lite Light weight PC (alias of pc-lite-2.6)
+
+    pc-lite-2.6Light weight PC
+
+Install the Docker bundle
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We can now install the `Clear Linux`_ bundle that containers Docker_:
+
+  ::
+
+    sudo swupd bundle-add opencontainers-dev
+
+Add your user to the KVM and Docker groups
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To enable your user to access both Docker and KVM you will need to add them to the relevant groups on the machine:
+ 
+  ::
+
+    sudo usermod -G kvm,docker -a <USERNAME>
+
+Download the clr-oci-runtime 2.0 code
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. attention::
+   Update this section when clr-oci-runtime is available in a Clear Linux bundle
+
+Download, build and install the clr-oci-runtime from source:
+
+  ::
+
+    cor_source=${HOME}/clr-oci-runtime
+    git clone https://github.com/01org/clr-oci-runtime.git $cor_source
+    cd $cor_source
+    autoreconf -fvi
+    bash autogen.sh --disable-cppcheck --disable-valgrind
+    make
+    sudo make install
+
+Restart Docker
+~~~~~~~~~~~~~~
+
+In order to ensure you are running the latest installed Docker_ you should restart the Docker_ daemon:
+
+  ::
+
+    sudo systemctl daemon-reload
+    sudo systemctl restart docker-upstream
+
+Final Docker sanity check
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before we dive into using `Clear Containers`_ it is prudent to do a final sanity check to ensure that relevant Docker_ parts have installed and are executing correctly:
+
+  ::
+
+    sudo systemctl status docker-upstream
+    docker-upstream ps
+    docker-upstream network ls
+    docker-upstream pull debian
+    docker-upstream run -it debian
+
+.. attention::
+   We should put the example output text in this section to aid clarity.
+
+If these tests pass then you have a working Docker_, and thus a good baseline to evaluate `Clear Containers`_ under.
+
+Enable Clear Containers runtime
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Now we have `Clear Containers`_ and Docker_ installed we need to tie them together by enabling the `Clear Containers`_ runtime within the Docker_ system:
+
+Locate where your OCI runtime got installed
+
+    ::
+
+      which clr-oci-runtime
+      #typically /usr/bin/clr-oci-runtime
+
+Then edit the Docker_ systemd unit file ExecStart to make `Clear Containers`_ the default runtime.
+
+  ::
+
+    Edit: /usr/lib/systemd/system/docker-upstream.service
+    ExecStart=/usr/bin/dockerd-upstream --add-runtime cor=/usr/bin/clr-oci-runtime--default-runtime=cor -H fd://
```

Comment:
Just so we get this clear - the action is to *add* arguments to the existing ExecStart line, yes? In which case the text needs to be a little clearer. Are you OK clarifying that @jodh-intel ?